### DISCUSSION TITLE
How do I manually (hard code) set Figure, Table, Section numbers?

### DISCUSSION THREAD

**Discussion by giorgiopaulon** [quarto-dev/quarto-cli#6400] [Feature Requests]
### Description

Is it possible to manually set the numbers of tables, figures or sections instead of using automatic numbering?

**Comment by rgaiacs** [comment]
@giorgiopaulon can you help me to understand your user case?

**Reply by giorgiopaulon** [reply]
I need to number tables and figures in a way that is not strictly related to the section where they appear, and it might even not be exactly sequential. In my document, the desired Table number sequence is 

2.1.1
2.1.2
2.2.1
2.2.2
3.1.1.1
3.1.1.2
3.1.2.1
3.1.2.2

**Comment by mcanouil** [comment]
Depending on the format you are using, you could use raw LaTeX or raw HTML, but let's be honest, that's not at all easy.

**Comment by cscheid** [comment]
Tracking the enhancement request [here](https://github.com/quarto-dev/quarto-cli/issues/6502). It's trickier than it looks (especially to support all formats) but it might be possible.
